File: shared/src/main/java/com/jawnnypoo/openmeh/shared/Story.java
package com.jawnnypoo.openmeh.shared;

import org.parceler.Parcel;

/**
 * Cool story brah
 */
@Parcel
public class Story {
    String title;
    String body;

    public Story(){}

    public String getTitle() {
        return title;
    }

    public String getBody() {
        return body;
    }
}

File: shared/src/main/java/com/jawnnypoo/openmeh/shared/model/Story.kt
package com.jawnnypoo.openmeh.shared.model

import com.squareup.moshi.Json

/**
 * Cool story brah
 */
class Story {
    @Json(name = "title") var title: String? = null
    @Json(name = "body") var body: String? = null
}
